WordPress/src/main/java/org/wordpress/android/ui/PagePostCreationSources.kt

```diff
@@ -0,0 +1,26 @@
+package org.wordpress.android.ui
+
+enum class PagePostCreationSources(val label: String) {
+    // post created from android shortcut long pressing on app icon
+    POST_FROM_SHORTCUT("post-from-short-cut"),
+    // post created from the posts list fab
+    POST_FROM_LIST("post-from-list"),
+    // page created from the pages list fab
+    PAGE_FROM_LIST("page-from-list"),
```

Comment:
I'm not sure these two are self-explanatory enough. What do you think about something like `page-from-pages-screen`?

Reply:
Makes sense...changed here 562213bef2f17d2322bc059bb2e00022122849b7